core/controlplane/config/templates/cloud-config-controller

```diff
@@ -106,7 +106,9 @@ coreos:
                   --mount volume=journal,target=/var/log/journal \
                   --volume machine-id,kind=host,source=/etc/machine-id,readOnly=true \
                   --mount volume=machine-id,target=/etc/machine-id \
+                  --uuid-file-save=/var/journald-cloudwatch-logs/journald-cloudwatch-logs.uuid \
```

Comment:
Can we use `/var/run/coreos/` instead of `/var/journald-cloudwatch-logs` for consistency?

Reply:
Perhaps we could replace all the `/var/run/coreos` to `/var/run/<svc>/` or `/var/run/kube-aws` as they aren't really tied to coreos but that would be an another github issue though!

Reply:
Whichever you prefer! I noticed there was an existing journald-cloudwatch-logs directory so put the file in there, but there's no reason why it can't be moved elsewhere.

Reply:
Thanks for confirming!
Yea, it seems like the directory is used for storing the state file according to [@jollinshead's setup](https://github.com/jollinshead/journald-cloudwatch-logs/blob/master/docker/init.sh#L6).
I don't have a strong opinion here so ;et's go ahead with the current directory.